appliance/uci_image.ks

```diff
@@ -93,6 +93,16 @@ echo "" >> /etc/ssh/sshd_config
 # Pull the latest UCI Image
 podman pull docker.io/manageiq/v2v-conversion-host:latest
 
+# Remove linux-firmware, installed by default
+yum -C -y --noplugins remove linux-firmware
+
+# Clean yum repodata
+yum --noplugins clean all
+
+# Zero out spaces
```

Comment:
Wouldn't `fstrim` provide the exact result? Can you compare?

Reply:
`fstrim` won't work as is (operation not supported), and some other changes will be needed in this kickstart file, on the build machine, or both. I'm not familiar with this area and have to do some digging...

Reply:
Ok, so let's leave it as it is then for now.